plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcSortItem.java

```diff
@@ -67,4 +67,14 @@ public int hashCode()
     {
         return Objects.hash(column, sortOrder);
     }
+
+    @Override
+    public String toString()
+    {
+        return new StringBuilder()
```

Comment:
Let's add sth like `io.trino.testing.BaseConnectorTest#testPredicateReflectedInExplain` for order by

- it either remains in the plan 
- or should be indicated in the TH